### ISSUE TITLE
JAWS reads the aria-describedby text on an input element 3 times

### ISSUE THREAD

**Issue by segedyjr** [FreedomScientific/VFO-standards-support#597]
### Summary

> Brief description of the issue. Please list any specific steps.
> 1. Open an HTML file with this code in a Chromium browser:
`<html><body><div><input aria-label="my field" id="id1" aria-describedby="id2" value="hello" /></div><div><label id="id2">My description.</label></div></body></html>`
> 2. Click into the input field

### Expected result

> JAWS would read the description "My description" one time.

### Actual result

> JAWS reads the description "My description" three times.

#### JAWS version and build number
JAWS Version 2021.2111.13 ILM

 
#### Operating System and version
Windows 10 Enterprise; version 20H2; OS build 19042.1348

#### Browser and version
Microsoft Edge version 96.0.1054.62, 64-bit

**Comment by JasonAment** [comment]
Using a <label> element for your description?

**Comment by segedyjr** [comment]
Fair point. I still experience the same problem when changing the `<label>` to a `<div>`

**Comment by StephenBeckettTPGi** [comment]
Haven't managed to replicate this issue in Edge (or FF or Chrome) running Windows 10. Will pass it over to a colleague to see how it behaves for them.

**Comment by frex65** [comment]
Bug confirmed in JAWS 2022.2112.24.

**Comment by menovak** [comment]
hey all,

this one caught my attention, so I thought I'd weigh in, please feel free to ignore if you've already solved this.

1 - I was not able to duplicate the original issue announce 'three times' (the aria-label and/or description)

2- I tested in Chrome 96 and Edge 96, using both JAWS 2021, and JAWS 2022 with default settings, on Windows 10, 64 bit

3 - I did find a difference between JAWS 2021 and JAWS 2022, in Edge, JAWS 2022 would sometimes announce twice (the aria-label and/or description) but never 3 times and it was not consistent

4 - To dig a bit deeper, I created a simple codepen, with the 1) original code, a 2) corrected version of the original code (as pointed out earlier, the LABEL element should not be used for the description text), and 3) an example using a visible label - not sure how long the codepen will exist but the URL is here - >   https://cdpn.io/menovak/debug/vYedQdz/vPMKKgLRLgqM

5 - Testing with the codepen, again using Chrome/Edge and  both JAWS 2021/2022 - the only place JAWS 2022 seems to announce twice (the aria-label and/or description) is what I would consider the incorrectly coded original version, and again is was not consistent.  JAWS 2021 did not trip on this.  

So it appears there is some difference in how JAWS 2022 treats the original code, versus JAWS 2021, but also considering I would flag the original code for incorrect use of the LABEL element, I'd say it is up to Ben to decide if worth investigating further.   

The other comment I would add is that using an input field without a visible text label in most cases is strongly discouraged - while there are exceptions, (e.g., the search field atop a web page with a typical magnifying glass  icon) i've not found many other good ones.

As always, feel free to let me know if I've miss-understood something...

Mark

**Comment by JAWS-test** [comment]
I tested the original code with JAWS 2022 and 2021 and found the following:
* JAWS 2022: The problem does not occur (not in Firefox, Chrome, Edge).
* JAWS 2021: The description is always output 3x, no matter if it is in a label element or a div element.

Note: The use of label for a description is not prohibited by the HTML and ARIA specification and thus should also be output correctly.
